### PR TITLE
[Tabs][Review and update content][1427069]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Formatting changes. Please review

### CODE REVIEW

**Review by KirtiPereira — CHANGES_REQUESTED** [review]

File: msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ ms.author: lajanuar
 ---
 # Create a configuration page
 
-A configuration page is a special type of [content page](content-page.md) that allows your users to configure some aspect of your Teams app. Typically these are used as part of:
+A configuration page is a special type of [content page](content-page.md) that allows your users to configure some aspects of Microsoft Teams app. It allows you to collect the information from your users and set the `contentUrl` of the content page to display. These aspects are used as part of the following:
```
[the remark this answers is not in the document]
Comment (Rajeshwari-v) [reply msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md]: Content page is in TOC under Tabs>how to > create tab pages > create a content page

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please do a thorough self-review.
Following is an example of what I see in the review page:
..., see [Mobile clients](./../tabs/how-to/create-tab-pages/configuration-page.md#Mobile clients).

**Review by Nikitha-MSFT — APPROVED** [review]
Reviewed and  approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**